test/prompt/invocation_layer/test_hugging_face.py

```diff
@@ -467,6 +467,14 @@ def test_stop_words_multiple_token(stop_words: List[str]):
     assert "good" not in result[0]
     assert "health" not in result[0]
 
+    if stop_words == ["unambiguously"]:
+        result = layer.invoke(
+            prompt="Generate a sentence `This is unrelated, but is ambiguously. So unambiguously.`",
+            stop_words=stop_words,
+        )
+        assert len(result) > 0
+        assert "So" in result[0]
+
```

Comment:
I think we can test the `StopWordsCriteria` directly in this case.
That way you can also mock the data without making any actual request as I don't think it's necessary to verify `StopWordsCriteria.__call__` behaviour.

Reply:
will do!